eth_defi/aave_v3/README.md

```diff
@@ -0,0 +1,84 @@
+# About Aave v3
+
+This README explains how Aave v3 interest calculation works and how we use the ReserveDataUpdated
+events to track interest rates.
+
+## Aave v3 blockchain events
+
+The ReserveDataUpdated event in [ReserveLogic.sol](https://github.com/aave/aave-v3-core/blob/master/contracts/protocol/libraries/logic/ReserveLogic.sol)
+(signature 804c9b842b2748a22bb64b345453a3de7ca54a6ca45ce00d415894979e22897a) is triggered whenever the interest rates of a reserve currency are updated.
```

Comment:
This is very good to keep documentation future-proof. I was thinking maybe use Aave core tags, so that you can see in the link what version it refers to: https://github.com/aave/aave-v3-core/blob/v1.16.2/contracts/protocol/libraries/logic/ReserveLogic.sol#L31